src/index.ts

```diff
@@ -1726,7 +1726,7 @@ function FlatpickrInstance(
           if (
             month.contains(elem) ||
             !(
-              m > 0 &&
+              m > self.config.showMonths - 1 &&
```

Comment:
This works, but I'm not sure if it's the correct fix, as `m > self.config.showMonths - 1` will always return false, making the if always execute

Reply:
Can you check the latest commits.
I think it fufils the original logic

Reply:
Which makes me think the logic is no longer required for multi month select due to the hidden class of the prevMonthDate and nextMonthDate